src/commands/shell.mjs

```diff
@@ -191,10 +199,13 @@ async function buildCustomEval(argv) {
           typecheck,
           performanceHints,
           format: outputFormat,
+          maxAttempts,
+          maxBackoff,
+          maxContentionRetries,
         });
 
         // If any query info should be displayed, print to stderr.
-        if (include.length > 0) {
+        if (include && include.length > 0) {
```

Comment:
is `include` ever undefined? curious how we'd get into that situation after telling yargs to parse include as an array.